ext/bcmath/tests/bug60377.phpt

```diff
@@ -6,8 +6,8 @@ if (PHP_INT_SIZE != 8) die("skip: 64-bit only"); ?>
 --FILE--
 <?php
 $var48 = bcscale(634314234334311);
-$var67 = bcsqrt(false);
-$var414 = bcadd(false,null,10);
```

Comment:
We will need some tests where the warning is actually thrown (preferably for all 4 instances where the warning can be thrown).

Reply:
> preferably for all 4 instances where the warning can be thrown

don't really understand what do you mean.

Reply:
There are four places in the code that will now throw a warning. There should be a test that checks these four warnings in some way.

Reply:
@nikic thanks for explanation